BioArchLinux/minimap2/hardening.patch

```diff
@@ -0,0 +1,45 @@
+--- minimap2-2.26.orig/Makefile	2023-04-29 21:51:09.000000000 +0530
++++ minimap2-2.26.new/Makefile	2023-10-21 13:31:34.438714294 +0530
+@@ -1,5 +1,6 @@
+-CFLAGS=		-g -Wall -O2 -Wc++-compat #-Wextra
+-CPPFLAGS=	-DHAVE_KALLOC
++CFLAGS+=		-g -Wall -fPIC -pie -O2 -Wc++-compat #-Wextra
++CPPFLAGS+=	-DHAVE_KALLOC
++LDFLAGS += -shared -pie
```

Comment:
`-shared` isn't needed here. It should be used only when linking the shared library.